src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkConnectorConfig.java

```diff
@@ -166,7 +169,13 @@ private static void addMetadataConfigs(ConfigDef configDef) {
                 ++orderInGroup,
                 Width.LONG,
                 "Datadog logs site"
-        );
+        ).define(
```

Comment:
I think we'll just need to add this to the README as well to document it.

Reply:
Added to README file